sdk/purestorageblock/arm-purestorageblock/package.json

```diff
@@ -0,0 +1,308 @@
+{
+  "name": "@azure/arm-purestorageblock",
+  "version": "1.0.0-beta.1",
+  "description": "A generated SDK for BlockClient.",
+  "engines": {
+    "node": ">=18.0.0"
+  },
+  "sideEffects": false,
+  "autoPublish": false,
+  "tshy": {
+    "exports": {
+      "./package.json": "./package.json",
+      ".": "./src/index.ts",
+      "./api": "./src/api/index.ts",
+      "./api/avsVmVolumes": "./src/api/avsVmVolumes/index.ts",
+      "./api/avsVms": "./src/api/avsVms/index.ts",
+      "./api/avsStorageContainerVolumes": "./src/api/avsStorageContainerVolumes/index.ts",
+      "./api/avsStorageContainers": "./src/api/avsStorageContainers/index.ts",
+      "./api/storagePools": "./src/api/storagePools/index.ts",
+      "./api/reservations": "./src/api/reservations/index.ts",
+      "./api/operations": "./src/api/operations/index.ts",
+      "./models": "./src/models/index.ts"
+    },
+    "dialects": [
+      "esm",
+      "commonjs"
+    ],
+    "esmDialects": [
+      "browser",
+      "react-native"
+    ],
+    "selfLink": false,
+    "project": "./tsconfig.src.json"
+  },
+  "type": "module",
+  "keywords": [
+    "node",
+    "azure",
+    "cloud",
+    "typescript",
+    "browser",
+    "isomorphic"
+  ],
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "files": [
+    "dist/",
+    "README.md",
+    "LICENSE",
+    "review/",
+    "CHANGELOG.md"
```

Comment:
I think the changelog tool issue has been closed? This two files are not required.